examples/aws-s3-folder/Main.scala

```diff
@@ -0,0 +1,88 @@
+import besom.*
+import besom.api.aws.*
+import besom.api.aws.s3.BucketObject
```

Comment:
Could we use the prefix notation like Pulumi consistently? I actually like it as it's very readable in Pulumi context, more than bare imported classes.

In this file I'd either import aws and use full aws.s3.BucketObject or import aws.s3 and use s3.BucketObject.

Reply:
You are actually using the s3.* syntax. Can you just use s3.BucketObject in the return type in the operation that maps files to BOs?

Reply:
sure, will do :)